desktop/package/package.gradle

```diff
@@ -113,6 +113,8 @@ task getJavaBinariesDownloadURLs {
             }
         }
 
+        ext.osKey = osKey
```

Comment:
Is this used? I'm not used to gradle, sorry.

Reply:
Yes, I need this to grab the right version of [appimagetools](https://github.com/AppImage/AppImageKit).

I use it later in `packageInstallers`:
```java
String osKey = getJavaBinariesDownloadURLs.property('osKey')
```

Reply:
Ah, okay.